katsdpcontroller/scheduler.py

```diff
@@ -2380,6 +2380,9 @@ def batch_tasks_skipped(self, n_tasks):
         """`n_tasks` batch tasks were skipped because a dependency failed."""
         pass
 
+    def batch_tasks_retried(self, n_tasks):
+        """`n_tasks` batch tasks failed and were re-scheduled."""
+
```

Comment:
Missing an explicit pass

Reply:
Thanks, added (it would actually have worked because the docstring functions like a `pass`, but it's inconsistent).